.travis.yml

```diff
@@ -2,19 +2,19 @@ language: bash
 services: docker
 
 env:
-  - VERSION=1.9 PYTHON_VERSION=2.7 PG_MAJOR=9.4 PG_VERSION=9.4.19-1.pgdg90+1 VARIANT=slim
```

Comment:
This is good.

As part of performing regular maintenance on this repository, we also check to see if there are any patch releases to dependencies within `X.Y/requirements.txt`.

I would reference existing PRs to see what normally changes during regular maintenance, and how we format the branch / PR body:

https://github.com/azavea/docker-django/pull/63

Reply:
Done in https://github.com/azavea/docker-django/pull/66/commits/b01a1df030f164ea0b2993b01828fd8c4f3244cd. Now that we're bumping `gevent` and `psycopg2` to the latest patches, too, there's quite a bit going on in this PR and it's hard to capture it in the title. Do you think we should accept a super long title and update it to include all of the versions that are getting bumped, or is it fine as is?